ruma-identity-service-api/src/tos/get_terms_of_service/v2.rs

```diff
@@ -0,0 +1,73 @@
+//! [GET /_matrix/identity/v2/terms](https://matrix.org/docs/spec/identity_service/r0.3.0#get-matrix-identity-v2-terms)
+
+use std::collections::BTreeMap;
+
+use ruma_api::ruma_api;
+use serde::{Deserialize, Serialize};
+
+ruma_api! {
+        metadata: {
+            description: "Gets all the terms of service offered by the server. The client is expected to filter through the terms to determine which terms need acceptance from the user. Note that this endpoint does not require authentication.",
+            method: GET,
+            name: "tos::get_terms_of_service::v2",
```

Comment:
```suggestion
            name: "get_terms_of_service",
```

It's not clear yet what's going to happen with the name field in the long run, but for consistency with the other endpoints, this shouldn't include the version or parent module name.

Reply:
Changed name.